saleor/webhook/response_schemas.py

```diff
@@ -0,0 +1,79 @@
+import logging
+from decimal import Decimal
+from typing import Any, cast
+
+from graphql.error import GraphQLError
+from prices import Money
+from pydantic import BaseModel, Field, field_validator
+
+from ..app.models import App
+from ..core.enums import CurrencyEnum
+from ..core.utils.metadata_manager import method_metadata_is_valid
+from ..graphql.core.utils import from_global_id_or_error
+from ..shipping.interface import ShippingMethodData
+from ..shipping.models import ShippingMethod
+from .const import APP_ID_PREFIX
+from .transport.shipping_helpers import to_shipping_app_id
+
+logger = logging.getLogger(__name__)
+
+name_max_length = ShippingMethod._meta.get_field("name").max_length
+
+
+class ShippingMethodSchema(BaseModel):
+    id: str | int
+    name: str = Field(..., max_length=name_max_length)
+    amount: Decimal = Field(..., ge=0)
+    currency: CurrencyEnum
+    maximum_delivery_days: int | None = Field(None, ge=0)
+    minimum_delivery_days: int | None = Field(None, ge=0)
+    description: str | None = None
+    metadata: dict[str, str] | None = {}
+
+    @field_validator("metadata", mode="before")
+    @classmethod
+    def clean_metadata(cls, value: Any) -> dict[str, str]:
```

Comment:
is value really Any? Isn't it `dict[str, str] | None`?

Reply:
The validation is run before the pydantic validation, so at this moment basically anything can be here